docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,172 @@
+
+Within the experimental section of the Spirit Design System Storybook, there is a
+"Feature Flags" folder. This contains stories and written documentation covering
+every available flag and how to configure them. Folders and stories within this
+section in the Storybook show components with all feature flags turned on.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the main package README file.
+
+Individual feater flags that are available for some component only will be also documented in
+the component's README file.
+
+## Expanding existing APIs
+
+New experimental functionality might sometimes be added with the need of an
+`UNSTABLE_` export or a feature flag. Oftentimes this ends up being a new prop
+on a component.
+The documentation associated with this prop (TypeScript types, Storybook controls, etc.) will state that it's experimental.
+
+## Moving to stable
+
+Over time it becomes apparent an experimental API has stabilized and suits the
+needs of most users. If it isn't a breaking change, and there hasn't been much
+movement, it can be moved from "experimental" status to be "stable".
+
+To initiate a move to stable, a new issue can be opened to request the team to evaluate
+if it can be moved (most of the time the team will initiate this move).
+The issue should contain the following criteria that need to
+be met to move a component from experimental to stable:
+
+- [ ] All components exported in `src/index.ts` and should not be `UNSTABLE_`
+      prefixed
+- [ ] Component should be documented on the website
+  - [ ] Component should have a usage, style, and code tab
+  - [ ] Component may have a component demo
+- [ ] For each component exported:
+  - [ ] Component is written as a function declaration or uses `forwardRef`
+  - [ ] Component has `types` defined
+    - [ ] Each prop type has a comment (used in Storybook)
+  - [ ] Default props are listed as default args in the function definition (not
+        in defaultProps)
+    - [ ] Note: default props should be stable, in other words props like
+          `onClick = () => {}` can cause re-renders since the function identity
+          is not stable
+  - [ ] Component has a story in `<ComponentName>.stories.ts`
+    - [ ] Component has an mdx document that follows our outline
+    - [ ] mdx document coverages at least common use-cases and provides a prop
+          table
+    - [ ] Stories cover at least common use-cases
+    - [ ] Stories may include a `Playground` story for controls
+      - [ ] Controls with no meaningful change to the component visuals should
+            be hidden from the controls panel, eg. `className`
+      - [ ] Props of type `node` with no proper controls available for
+            configuration should be hidden from the controls panel, eg.
+            `children`
+    - [ ] Stories should mirror intended usage of the component
+  - [ ] Component has unit/integration tests written in RTL for testing the
```

Comment:
(I read this as Right-to-left and was bit confused :D)

Reply:
Fixed.